solidity/deploy/07_disable_redemptions.ts

```diff
@@ -0,0 +1,40 @@
+import type { HardhatRuntimeEnvironment } from "hardhat/types"
+import type { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async (hre: HardhatRuntimeEnvironment) => {
+  const { deployments, ethers, getNamedAccounts } = hre
+  const { execute, read } = deployments
+  const { deployer } = await getNamedAccounts()
+
+  deployments.log("disabling redemptions in the Bridge")
+
+  // To disable redemptions, we need to make the redemption dust threshold
+  // extremely high. The redemption dust threshold value is determined by the
+  // redemptionDustThreshold governance parameter. This parameter is uint64.
+  // The maximum value for the uint64 is 2^64-1 = 18446744073709551615.
+  const redemptionDustThreshold = ethers.BigNumber.from("18446744073709551615")
+
+  // Fetch the current values of other redemption parameters to keep them unchanged.
+  const redemptionParameters = await read("Bridge", "redemptionParameters")
+
+  await execute(
+    "Bridge",
+    { from: deployer, log: true, waitConfirmations: 1 },
+    "updateRedemptionParameters",
+    redemptionDustThreshold,
+    redemptionParameters.redemptionTreasuryFeeDivisor,
+    redemptionParameters.redemptionTxMaxFee,
+    redemptionParameters.redemptionTxMaxTotalFee,
+    redemptionParameters.redemptionTimeout,
+    redemptionParameters.redemptionTimeoutSlashingAmount,
```

Comment:
Per _Adjust parameters for Chaosnet phase 1_ section of https://github.com/keep-network/tbtc-v2/issues/438, let's set the slashing amount to 0.

Reply:
See d372066d